workshops/scrapbook_css/README.md

```diff
@@ -0,0 +1,263 @@
+
+#### Hacking
+
+It's time for you to control, there are so many epic things you can do with CSS! So go forward and make your Scrapbook unique to you!! Here a few guiders:
+
+- Make a gradient background. You won't be able to do this with the CSS variables but you can by applying the background to the body tag instead. I'm saying too much, learn more about CSS gradients [here](https://css-tricks.com/css3-gradients/)!
+- You can change the text of the UI. Check out this [article](https://css-tricks.com/swapping-out-text-five-different-ways/#css-only-way) for to get a better sense of what I'm talking about! That's what I call a hack ;)
+- Animate everything! You've seen how we can animate elements in CSS but we've really only changed the colour and made something bigger. There are so many more properties that are animatable, check out this [extensive list](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties). Also, check out my classes reference just below so you know how to change each element on the page.
+
+### Publishing
+
+To publish click `Add to Gist`. Copy the link. Now hop over to the #scrapbook-css channel in Slack and paste the link into a new message to apply it to your scrapbook!
+
+Once you're done making your profile pretty, how about sharing it in #scrapbook? Tag me (@sampoder), I'd love to see it!
+
+#### Reference Guide
+
+Here are all the classes you'll need to know to make a splendid Scrapbook:
+
+`header` is the entire section above your posts.
+
+`header-col-1` is the section with your avatar, name, streak (if set to display), social links and profile audio (if set). 
+
+`header-title-avatar` is your Avatar, it is a `<img>`.
+
+`header-title-name` is the heading with your name.
+
+`header-content` contains all of your social links and your streak (if set to display).
+
+`header-streak` is the indicator of your streak.
+
+`header-links` are your social links.
+
+`header-link` is a social link, there are likely multiple in the links section.
+
+`header-webring` is a section that contains each person in your webring if you've created one.
+
+`header-webring-mention` is a mention of a person in your webring, there are likely multiple in the webring section.
+
+`header-chart` is the chart that shows how often you post.
+
+`posts` is the section with all your posts.
+
+`post` is a post.
+
+`post-header` is the part of the post with the date / time.
+
+`post-attachments` is the section with each image, video or audio file you've attached to your Scrapbook post.
+
+#### Inspiration
+
+To finish off, I'd like to leave you with some amazing customized Scrapbook profiles.
+
+- The god of Scrapbook profiles: Caleb's scrapbook is too amazing for me to describe. Honestly, just [check it out](https://scrapbook.hackclub.com/caleb/).
+- Ghoshwolf has made a retro-themed Scrapbook, view it [here](https://scrapbook.hackclub.com/pauliukas.gec).
```

Comment:
```suggestion
- Ghoshwolf has made a [retro-themed Scrapbook](https://scrapbook.hackclub.com/pauliukas.gec).
```
It’s much easier to scan a site & find what you’re looking for (& especially helpful to screenreader users) to link text that describes what the link is, instead of text like “here” or “link.” This isn’t a big deal, but I think it’s both easier to read & scan this way